hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -1058,7 +1065,7 @@ private int getMostLocalEntityForRegion(int region) {
     }
 
     private double getWeightedLocality(int region, int entity) {
-      return cluster.getOrComputeWeightedLocality(region, entity, type);
+      return cluster.getOrComputeWeightedLocality(region, entity, type, onlyOnSsd);
```

Comment:
Rather than a boolean for ssd, should this be a type? Maybe we want to favor non-ssd in another balancer impl?

Reply:
fix